CHANGELOG.md

```diff
@@ -132,6 +132,7 @@ The types of changes are:
 * Adds endpoint analytics events [#622](https://github.com/ethyca/fidesops/pull/622)
 * Sample dataset for Salesforce with access configuration [#676](https://github.com/ethyca/fidesops/pull/676)
 * Sample dataset and access configuration for Zendesk (ticket endpoints) [#677](https://github.com/ethyca/fidesops/pull/677)
+* Sample dataset for Sendgrid with access configuration [#883](https://github.com/ethyca/fidesops/pull/883)
```

Comment:
@HamzaWaseemOnBench please follow the instructions provided in the PR template that describe how to update the `CHANGELOG.md` file. there are a few things you need to do here:
- merge in `main` so the most recent CHANGELOG.md file is being appended to. this is to avoid merge conflicts in the `CHANGELOG.md` file.
- your description for this feature needs to be added in the `Unreleased` section, since you are not adding this feature to the `1.6.0` release, but you're adding it to the next upcoming release.  You can also refer to the [Keep a Changelog](https://keepachangelog.com/en/) documentation.
- the description you gave here doesn't really capture the full scope of your changes - i think a more accurate description would read something like: "Access and erasure support for Sendgrid `contacts` endpoint"